subsys/zigbee/common/zigbee_ota.h

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef ZIGBEE_OTA_H__
+#define ZIGBEE_OTA_H__
+
+#define ZB_HA_DEVICE_VER_OTA_UPGRADE_CLIENT        0
+#define ZB_HA_OTA_UPGRADE_CLIENT_DEVICE_ID         0xfff0
+
+/* OTA Upgrade client test input clusters number. */
+#define ZB_HA_OTA_UPGRADE_CLIENT_IN_CLUSTER_NUM    1
+/* OTA Upgrade client test output clusters number. */
+#define ZB_HA_OTA_UPGRADE_CLIENT_OUT_CLUSTER_NUM   1
+
+
+/**
+ *  @brief Declare cluster list for OTA Upgrade client device.
+ *  @param cluster_list_name [IN] - cluster list variable name.
+ *  @param basic_attr_list [IN] - attribute list for Basic cluster.
+ *  @param ota_upgrade_attr_list [OUT] - attribute list for OTA Upgrade client
+ *                                       cluster
+ */
+#define ZB_HA_DECLARE_OTA_UPGRADE_CLIENT_CLUSTER_LIST(                         \
```

Comment:
Can we format these macros to be a bit more readable?

```suggestion
#define ZB_HA_DECLARE_OTA_UPGRADE_CLIENT_CLUSTER_LIST(cluster_list_name,       \
						      basic_attr_list,         \
						      ota_upgrade_attr_list)   \
	zb_zcl_cluster_desc_t cluster_list_name[] =                            \
	{                                                                      \
		ZB_ZCL_CLUSTER_DESC(                                           \
			ZB_ZCL_CLUSTER_ID_BASIC,                               \
			ZB_ZCL_ARRAY_SIZE(basic_attr_list, zb_zcl_attr_t),     \
			(basic_attr_list),                                     \
			ZB_ZCL_CLUSTER_SERVER_ROLE,                            \
			ZB_ZCL_MANUF_CODE_INVALID),                            \
		ZB_ZCL_CLUSTER_DESC(                                           \
			ZB_ZCL_CLUSTER_ID_OTA_UPGRADE,                         \
			ZB_ZCL_ARRAY_SIZE(ota_upgrade_attr_list,               \
					  zb_zcl_attr_t),                      \
			(ota_upgrade_attr_list),                               \
			ZB_ZCL_CLUSTER_CLIENT_ROLE,                            \
			ZB_ZCL_MANUF_CODE_INVALID)                             \
	}
```

Reply:
I agree with your proposal. This gives a better separation between macro parameters and body.